website/blog/2018-06-19-single-spa-parcels-explained.md

```diff
@@ -0,0 +1,70 @@
+
+For those familiar with [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) and [custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements), you may be wondering why a javascript library would try to do what browsers are starting natively to do.
+
+And as one of the contributors to the custom elements polyfill, let me be the first one to say that we did not make this decision lightly.
+
+If you’re interested in diving into the details, check out [One Company’s Relationship With Custom Elements](https://medium.com/canopy-tax/one-companys-relationship-with-custom-elements-d360baf3b253), which explains some of the difficulties we’ve been through with web components and custom elements.
+
+TLDR: React and some other frameworks don’t interop with custom elements very well. Additionally dealing with inner html, attributes vs properties, and customized builtins can be a pain.
+
+## Okay but you haven’t told me what a single-spa parcel is
+
+A parcel is single-spa’s way of building a component in one framework and using it in another.
+
+To implement a parcel, just create a javascript object that has 3–4 functions on it. We call this javascript object a *parcel config* and there are three required functions to implement: bootstrap, mount, and unmount. A fourth function, update, is optional.
+
+Each of the functions will be called by single-spa at the right time, but the parcel config will control what happens. In other words, single-spa controls the “when,” but the parcel config controls the “what” and the “how.”
+
+Once you’ve implemented the parcel config, simply call singleSpa.mountRootParcel(parcelConfig, parcelProps) to mount it. This is the key to what makes parcels framework agnostic — regardless of whether the parcel config is implemented with React, Angular, Vue, or anything else, to use the parcel you always just call mountRootParcel().
+
+## A few more specifics
+
+We’ve glossed over a few things that I want to touch on real quick:
+
+* **How do you implement the lifecycle functions on the parcel config?**
+
+		Use a helper library for your framework of choice. [single-spa-react](https://github.com/CanopyTax/single-spa-react), [single-spa-angular2](https://github.com/CanopyTax/single-spa-angular2) (for angular@2+), [single-spa-angular1](https://github.com/CanopyTax/single-spa-angular1), [single-spa-vue](https://github.com/CanopyTax/single-spa-vue), and [others](https://github.com/CanopyTax/single-spa/blob/master/docs/single-spa-ecosystem.md) will implement the entire parcel config for you.
+
+* **What are the props you pass to mountRootParcel()?**
+
+		The props passed as the second argument to singleSpa.mountRootParcel(parcelConfig, parcelProps) are an object with one required prop and as many custom props as you’d like. The required prop is domElement, which tells the parcel where to mount. And the custom props get passed through to the parcel config lifecycle functions.
```

Comment:
"is an object" vs "are an object".
I might also recommend rewriting the second portion of that sentence to make it easier to read. 
"... is an object which requires one prop, but can take as many custom props as you'd like to add."